PyPowerFlex/base_client.py

```diff
@@ -370,3 +370,17 @@ def _query_selected_statistics(self, action, params=None):
             LOG.error(exc.message)
             raise exc
         return response
+
+    def _modify_entity(self, action, sdc_id, params):
```

Comment:
I did not find the usage of this method, Where it has been used?

Reply:
Forgot to remove this method. No need to add extra function in base_client.py. Removed.